### PR TITLE
[WM-2185] render noncreator error card when user can't create cromwell

### PR DESCRIPTION
**Pull request by mspector**
### What
- When a user in an Azure workspace is not the creator of that workspace, an error message should appear in the Workflows tab. That error message is the same as the one displayed in the Cloud Environment sidebar modal.

### Why
- Until "multi-user Cromwell" is enabled, only the workspace creator should be able to access Cromwell. The logic that determined the content of the Workflows tab was faulty, causing workspace non-creators to see the "Workflow App Launcher" card when they should have seen this error instead.

### Testing strategy
Manual testing only. This is a temporary fix to an experience that will soon be revamped entirely.

 
image

### CODE REVIEW

File: src/workflows-app/SubmitWorkflow.js [hunk 1]
```diff
@@ -126,51 +127,55 @@ export const SubmitWorkflow = wrapWorkflowsPage({ name: 'SubmitWorkflow' })(
       }
     });
 
-    return loading
-      ? centeredSpinner()
-      : div([
-          !pageReady && h(WorkflowsAppLauncherCard, { onClick: createWorkflowsApp, disabled: launcherDisabled }),
-          pageReady &&
-            div({ style: { margin: '2rem 4rem' } }, [
```
Comment (mspector) [inline src/workflows-app/SubmitWorkflow.js:134]: to make this diff more obvious: I moved this `div` (the main contents of the SubmitWorkflow page) into its own function, `renderSubmitWorkflow` (see added code below), and moved the flow control to a `Utils.cond` statement.

**Comment by nawatts** [comment]
Out of curiosity, could the information in the tooltip be rendered in place of the disabled button? Instead of requiring a user to hover over the button to access it.

File: src/workflows-app/utils/app-utils.js [hunk 2]
```diff
@@ -7,6 +7,9 @@ import { AppProxyUrlStatus, getUser, workflowsAppStore } from 'src/libs/state';
 import * as Utils from 'src/libs/utils';
 import { cloudProviderTypes } from 'src/libs/workspace-utils';
 
+export const getCromwellUnsupportedMessage = () =>
```
Comment (jdcanas) [inline src/workflows-app/utils/app-utils.js:10]: nit: seems like this should be a static const string and the call site can wrap it in the function if needed. Not super opinionated about this though.

**Review by jdcanas — APPROVED** [review]

File: src/workflows-app/utils/app-utils.js (hunk 2 above)
Comment (mspector) [reply src/workflows-app/utils/app-utils.js:10]: yeah. at the time I was thinking I'd leave room to make it parameterized or something. but I see it's not really helpful here

**Review by salonishah11 — APPROVED** [review]

**Comment by mspector** [comment]
> Out of curiosity, could the information in the tooltip be rendered in place of the disabled button? Instead of requiring a user to hover over the button to access it.

@nawatts , I think that'd be easy enough, but it feels like just enough of a user-facing change that I'd want to consult with our PM first

**Pull request merged**